docs/changelog.md

```diff
@@ -5,12 +5,30 @@ toc:
 
 # Redocly CLI changelog
 
+## 1.0.0-beta.130 (2023-07-06)
+
+### Features
+
+- Added new rule `component-name-unique` to check for unique component names (kudos to @pboos 🎉).
+- Added docker container to GitHub packages.
+
+### Fixes
+
+- Fixed an issue when the `--files` option of the `push` command did not upload extra files.
+- Fixed wrong schema type validation.
```

Comment:
What did we fix here? Was this the null validation thing? Can we expand a bit please

Reply:
@lornajane how about this?
```suggestion
- Fixed nullable schema type validation for OAS 3.1.
```